OMEdit/OMEditLIB/Util/Helper.cpp

```diff
@@ -53,6 +53,7 @@ QString Helper::omnotebookFileTypes = "OMNotebook Files (*.onb *.onbz *.nb)";
 QString Helper::ngspiceNetlistFileTypes = "ngspice Netlist Files (*.cir *.sp *.spice)";
 QString Helper::imageFileTypes = "SVG (*.svg);;PNG image (*.png);;Windows BMP image (*.bmp);;TIFF (*.tiff)";
 QString Helper::bitmapFileTypes = "Image Files (*.png *.bmp *.jpg *.jpeg);;PNG image (*.png);;Windows BMP image (*.bmp);;JPEG (*.jpg *.jpeg)";
```

Comment:
the "bitmapFileTypes" are used also for texturing selection in 3D animation viewer, so not extending this one with SVG as it's not the intention of this PR.